### ISSUE TITLE
Subject picker styling issues

### ISSUE THREAD

**Issue by chugginselifesciences** [libero/reviewer#1346]
In Reviewer the subject picker has too little padding for the placeholder text, and the keywords are too small.
![image](https://user-images.githubusercontent.com/41907215/91424972-6e142e00-e852-11ea-852b-3fe2ede4c1b6.png)
![image](https://user-images.githubusercontent.com/41907215/91425060-8e43ed00-e852-11ea-83b7-88e8b78fd8cc.png)

In xPub:

![image](https://user-images.githubusercontent.com/41907215/91425021-808e6780-e852-11ea-97c8-76331c799fb4.png)
![image](https://user-images.githubusercontent.com/41907215/91425034-85531b80-e852-11ea-98a7-3d50165f1b25.png)

____________

To fix this it looks like we could add 0.375rem/6px padding to the left (removing the 1.25rem top, bottom and right padding) of the input and changing the margin on the keyword/subject 'capsules' from 3px/.125rem to just 6px/0.375rem right margin.

![image](https://user-images.githubusercontent.com/41907215/91425312-e1b63b00-e852-11ea-9c00-bc944d8e8460.png)

![image](https://user-images.githubusercontent.com/41907215/91426208-1971b280-e854-11ea-86d6-937c44ed99c0.png)

In summary I think we should:

- [ ] Change padding of `.subject-area .select-field__value-container` to `0 0 0 0.375rem`
- [ ] Change margin of `.select-field__input .select-field__multi-value` to `0 0.375rem 0 0`
- [ ] Font size of `.select-field__input .select-field__multi-value .select-field__multi-value__label` should be 16px/1rem, which makes the keywords the correct height (36px). 
- [ ] The 'x' is too small but I can't tell where it's coming from. Is it SVG?
Please let me know if you need an SVG or please take from xPub.
